test/controllers/predicates/AuthorisedActionSpec.scala

```diff
@@ -295,10 +295,10 @@ class AuthorisedActionSpec extends TestSuite {
 
           lazy val result = {
             mockAuthReturnException(NoActiveSession)
-            auth.async("1234567890")(block)
+            auth.async(block)
           }
 
-          status(result(fakeRequest)) mustBe UNAUTHORIZED
+          status(result(fakeRequestWithMtditid)) mustBe UNAUTHORIZED
         }
```

Comment:
could you add the following unit test so we get coverage on the new execution path added in the AuthorisedAction

        "the request does not contain an mtditid header" in {

          lazy val result = auth.async(block)

          status(result(FakeRequest())) mustBe UNAUTHORIZED
        }